abc_bench.py

```diff
@@ -0,0 +1,56 @@
+import csv
```

Comment:
This file was temporarily added for sharing purposes. It will removed in a future commit.